app/assets/javascripts/controllers/report_data_controller.js

```diff
@@ -198,6 +200,15 @@
     this.initController(this.initObject);
   };
 
+  ReportDataController.prototype.gtlSetOneRowActive = function(item, event) {
+    this.gtlData.rows.map(row => { row.selected = false });
```

Comment:
This is the reason all the ruby specs fail...

Arrow functions are **not** allowed in the asset pipeline ;)

Reply:
(indeed, ruby specs are now green, when using `this.gtlData.rows.map(function(row) {`

Reply:
Fixed, lets hope Travis will agree